spec.md

```diff
@@ -1616,12 +1617,37 @@ A register is initialized with an indeterminate value (see [@sec:indeterminate-v
 
 # Conditionals
 
-Several statements provide branching in the data-flow and conditional control of verification constructs.
+Conditional statements define one or more regions where the operations (declarations, statements, and expressions) inside these regions *conditionally execute* based on the value of a condition.
```

Comment:
> Each conditional statement defines a regions. Statements and expressions inside this region execute conditionally based on the value of an expression. Declarations are also allowed inside of conditional statements, but the resulting hardware component always exists.

Trying to figure out how to suggest an edit, but I fail. This verbiage is a little more exact for this sentence.